hive/src/main/java/com/netease/arctic/hive/op/UpdateHiveFiles.java

```diff
@@ -261,6 +268,30 @@ private void checkCreatePartitionDataFiles(List<DataFile> addFiles, String parti
     }
   }
 
+  /**
+   * check files in the partition, and delete orphan files
+   */
+  private void checkOrphanFilesAndDelete() {
+    List<String> partitionsToCheck = this.partitionToCreate.values()
+        .stream().map(partition -> partition.getSd().getLocation()).collect(Collectors.toList());
+    for (String partitionLocation: partitionsToCheck) {
+      if (!table.io().isEmptyDirectory(partitionLocation)) {
```

Comment:
why call this method?  use `table.io.list` instead.  
don't call the unnecessary hadoop rpc.